src/main/webapp/sass/_core.scss

```diff
@@ -3450,6 +3453,12 @@ div.border {
 }
 
 #body .profileContainer {
+	.profileLinksContainer {
```

Comment:
Moved some style from SRC to SWC.  This styles the container that is used to hold the new User Links component